brightnessctl.1

```diff
@@ -51,7 +51,7 @@ Display value as a percentage in get.
 .sp
 \fB\-n, \-\-min\-value\fP=\fIVALUE\fP
 .RS 4
-Set minimum brightness when using delta values, defaults to 1.
+Set minimum brightness when using delta values.
```

Comment:
I'd rather leave it as it is and change the current minimum instead, since `0` have undefined behaviour in different setups as it seems, either minimal brightness or completely shutdown the backlight.

Reply:
This is not necessarily undefined behavior. It is just behavior that is not defined by brightnessctl. The display turning off may be entirely desirable and intended behavior by the device manufacturer. It is not as if this is not configurable. `0` seems like a more sensible default. If that default is undesirable for a specific use case, the `-n` flag is there.

Reply:
I'm talking from the user's standpoint, on whos system `0` turns backlight off which means completely black screen. And it makes impossible for me (by default) to reduce the brightness to a minimum using -5% steps (for example), because I can't see anything at all when backlight is off, it's basically as if display is off.

Reply:
I'm assuming that the behavior of your device is less common, although I have no actual data to back this up.

Consider the behavior for a device that does not turn off at `0`. The default behavior would mean the limited minimum is `1` greater than the minimum that the device is capable of. A user of such a device would need to specify `-n 0` to get the actual minimum brightness. This may be a small difference that most probably would not notice, but it is odd behavior.

For a device that is capable of turning off at a brightness of `0`, why should brightnessctl arbitrarily limit this capability?

Reply:
To have consistent behaviour on different devices, i.e. not turning off screen unexpectedly on some.

Reply:
Different devices can have different capabilities. brightnessctl should not limit the capabilities of a device for the sake of consistency across devices. I have already given the reasons for why a minimum of `1` does not make sense as a default. If the default minimum brightness behavior is not what you personally desire, you can change it with the `-n` or `--min-value` flags.